media-libs/oidn/oidn-1.3.0.ebuild

```
@@ -34,6 +34,8 @@ DEPEND="
 
 CMAKE_BUILD_TYPE=Release
 
+PATCHES="${FILESDIR}/glibc-1.3.0.patch"
```

Always treat PATCHES as an array:

```suggestion
PATCHES=( "${FILESDIR}"/glibc-1.3.0.patch )
```

Note that we normally name patches as ${PN}-versionAppliedToFirst-description.patch.

Looks good to me, although I'll have to mirror the patch on my end as it's a bit large (not your fault! It's best if I do that for now.)

---

Renamed the file with the suggested naming scheme :)